### DISCUSSION TITLE
Problema con Task 2: Group statistics

### DISCUSSION THREAD

**Discussion by cristianpenagos** [rramosp/ai4eng.v1#299] [02 - PYTHON]
En este ejercicio mi código es el siguiente:

def get_stats(df):
   
    df = df.copy()
    stats = df.groupby('category')['price'].agg(['mean', 'max', 'min'])
    stats.columns = ['media', 'maximo', 'minimo']
    #stats.reset_index(inplace=True)
  
    return stats
    
    
al hacer las pruebas con el código arroja el data frame con los valores esperados pero al enviar al rlxmooc aparace el siguiente mensaje: 

task_02 submitted. your grade is 0
----- grader message -------
incorrect:
testing with 100 random calls
index must be of type int

input dataframe is
         price  category  margin
itemid                          
18479   130.59       2.0    9.39
89485    85.93       0.0   10.30
31761    78.53       2.0    9.92
48257   112.64       0.0   10.84
3215    108.48       2.0   10.74
92501    94.96       1.0   10.51
32377    79.62       1.0    9.39
74739    99.37       1.0    9.38
97777    97.57       0.0    8.62
78992    91.23       2.0   11.04
43919    94.07       0.0    8.89
57155   113.11       0.0    9.32
83211   100.81       2.0    8.83
19063    95.36       0.0   10.87
51532   102.59       2.0   11.13
41154   105.61       0.0    9.81
87789    95.24       2.0    8.88
5709     97.77       1.0    8.16
16607   114.06       2.0    8.57
30033    82.34       1.0   11.17
3036    109.02       1.0   10.82
74387    96.29       1.0   12.01
47983   105.49       1.0    9.41
41590    89.71       2.0    9.80
2568     93.59       0.0    9.68

expected output was
                media  maximo  minimo
categoria                            
0           99.735000  113.11   85.93
1           95.607500  109.02   79.62
2          101.248889  130.59   78.53

but obtained
               media  maximo  minimo
category                            
0.0        99.735000  113.11   85.93
1.0        95.607500  109.02   79.62
2.0       101.248889  130.59   78.53

Nótese que los datos esperados y los obtenidos son los mismos, sin embargo la calificación es 0.

**Comment by Felipe-RA** [comment]
Hola @cristianpenagos 

Ya verificaste el indice? Debe ser también del mismo tipo.

Veo que el tuyo es un float, pero se espera un int. 

```python
index must be of type int
```
